### PR TITLE
Updated the PrestaShop documentation slightly

### PR DESCRIPTION
**Pull request by BitcoinMitchell**

### CODE REVIEW

**Comment by BitcoinMitchell** [comment]
The first few times I messed up, but for reason it fails to build a proper link to the new image (even though it works for actual images on the page). If anyone has any ideas, please tell me (or feel free to edit the PR itself).

**Comment by ndeet** [comment]
copy from mattermost:

Seems you made a normal link, images have a ! in front. Try 
Invalid Token-popup 

If it does not work, try to remove your backticks around "Invalid Token"-popup

File: docs/PrestaShop.md [hunk 1]
```diff
@@ -46,7 +48,7 @@ No matter if you're using a self-hosted or third-party solution from step 2, the
 
 ![BTCPay Server PrestaShop API key setup](./img/prestashop/api-key-setup.jpg)
 
-7. Press the `Authorize app`-button after which you will be redirected back to your Prestashop store.
+7. Press the `Authorize app`-button after which you will be redirected back to your Prestashop store. If you get an [`Invalid Token`-popup](./img/prestashop/invalid-token-popup.jpg), please make sure that PrestaShop and BTCPay Server both use HTTPS and have proper hostnames (see [Server Requirements](#server-requirements)).
```
Comment (dennisreimann) [inline docs/PrestaShop.md:51]:
```suggestion
7. Press the `Authorize app`-button after which you will be redirected back to your Prestashop store. If you get an "Invalid Token"-popup, please make sure that PrestaShop and BTCPay Server both use HTTPS and have proper hostnames (see [Server Requirements](#server-requirements)). ![Invalid Token](./img/prestashop/invalid-token-popup.jpg)
```

**Review by pavlenex — APPROVED** [review]

**Comment by BitcoinMitchell** [comment]
I actually wanted it as a link, but I guess showing the image is even better. Thanks for merging! 😁

**Comment by ndeet** [comment]
Ah, got it. Then likely the backticks in side the [] where the problem but @dennisreimann fixed it anyway and showing the image is easier for the user too I think.

**Pull request merged**